app/frontend/courses/curricula/types/CoursesCurriculum__QuizQuestion.res

```diff
@@ -42,14 +42,14 @@ let lastQuestion = questions => {
 
   questions |> ArrayUtils.unsafeFind(
     q => q.index == maxIndex,
-    "Could not find last question at index " ++ string_of_int(maxIndex),
+    I18n.t("shared.last_question_not_found") ++ string_of_int(maxIndex),
```

Comment:
`last_question_not_found` is not reused; It should not be part of the shared group